geometry_msgs/Point32[] corners
CollisionCircle[] collision_circles